dags/sources/bigquery.py

```diff
@@ -59,20 +64,24 @@ def get_data(
     query = (
         f"SELECT *"
         f" FROM `{self.location}`"
+        f" ORDER BY {self.unique_id}"
         f" LIMIT {limit} OFFSET {offset}"
     )
     query_job = self.client.query(query)
 
-    rows = []
-    for element in query_job.result():
-      # create dict to hold results and respect the return type
-      row = {}
-      for f in fields:
-        if f in element.keys():
-          row[f] = element.get(f)
-      rows.append(row)
+    try:
```

Comment:
Why does the loop require a "BadRequest" try/catch, but the self.client.query doesn't? Was there a special situation that occurs with BigQuery client here?

Reply:
The actual call to BQ is done upon calling query_job.result().

But I see that this is not clear the way it is written, I'll add a explicit variable to make it obvious.

BadRequest is being throw when one tries to ORDER BY using a field that does not exist in the table. I'll also add a comment to that.